### DISCUSSION TITLE
Add shorts video to playlist

### DISCUSSION THREAD

**Discussion by RobinRMC** [ReVanced/revanced-discussions#123] [Question]
YouTube doesn't show you the option to add a shorts video to a playlist, while this is possible when you play it in the normal player.

I think that it would be great if the button to add the video to a playlist would also show in shorts.

**Comment by qb20nh** [comment]
Opening shorts as normal video could make everything so simple

**Reply by RobinRMC** [reply]
Yes, but that should be another feature.

**Reply by LisoUseInAIKyrios** [reply]
@RobinRMC there already is an open issue for playing shorts like regular videos.

This discussion is very old and feature requests are no longer here, so the discussion was closed as stale.

**Reply by RobinRMC** [reply]
Thank you for clarifying @LisoUseInAIKyrios.

If I understand you correctly, if I specifically want a button in the shorts player to add the short to a playlist, I would have to create an issue on the patches repository?

**Reply by LisoUseInAIKyrios** [reply]
Correct.  But first search thru all issues there (both open and closed), because someone may have already created an issue.
